spec/src/main/asciidoc/javaee/el.asciidoc

```diff
@@ -1,46 +0,0 @@
-[[el]]
-== Integration with Unified EL
-
-[[el_resolution]]
-=== Bean name resolution in EL expressions
-
-The container must provide a Unified EL `ELResolver` to the servlet engine and JSF implementation that resolves bean names using the rules of name resolution defined in <<name_resolution>> and resolving ambiguities according to <<ambig_names>>.
-
-* If a name used in an EL expression does not resolve to any bean, the `ELResolver` must return a null value.
-* Otherwise, if a name used in an EL expression resolves to exactly one bean, the `ELResolver` must return a contextual instance of the bean, as defined in <<contextual_instance>>.
-
-[[el_support]]
-=== Unified EL integration API
-
-Since CDI version 4.1, the Unified EL integration API, which is part of the `BeanManager` API, is deprecated.
-The relevant methods are placed in a new interface `jakarta.enterprise.inject.spi.el.ELAwareBeanManager`, which is present in a new supplemental CDI API artifact: `jakarta.enterprise:jakarta.enterprise.cdi-el-api`.
-
-==== Obtaining `ELAwareBeanManager`
```

Comment:
Can we actually remove this file?
Technically the `ELAwareBeanManager` is still an API that resides in [CDI repo](https://github.com/jakartaee/cdi/blob/main/el/src/main/java/jakarta/enterprise/inject/spi/el/ELAwareBeanManager.java).

Reply:
That is discussion servlet container integration requirements so that should move the the platform spec. We should add a note on the deprecation to the core spec.